content/_data/upgrades/2-235-3.adoc

```diff
@@ -24,3 +24,46 @@ Update Red Hat compatible operating systems (Red Hat Enterprise Linux, CentOS, F
 ----
 
 See the link:/blog/2020/07/27/repository-signing-keys-changing/[blog post] for more details.
+
+==== New Windows Installer
```

Comment:
I would rather move it to .4. Nobody reads changelogs retrospectively.

Reply:
Done in b90e4d48823ba4b8c66a2be24981cb260c38e980